plugins/inputs/monit/monit.go

```diff
@@ -0,0 +1,393 @@
+			fields["response_time"] = service.Port.ResponseTime
+			fields["protocol"] = service.Port.Protocol
+			fields["type"] = service.Port.Type
+			acc.AddFields("remote_host", fields, tags)
+		} else if service.Type == "5" {
+			fields["cpu_system"] = service.System.CPU.System
+			fields["cpu_user"] = service.System.CPU.User
+			fields["cpu_wait"] = service.System.CPU.Wait
+			fields["cpu_load_avg_1m"] = service.System.Load.Avg01
+			fields["cpu_load_avg_5m"] = service.System.Load.Avg05
+			fields["cpu_load_avg_15m"] = service.System.Load.Avg15
+			fields["mem_kb"] = service.System.Memory.Kilobyte
+			fields["mem_percent"] = service.System.Memory.Percent
+			fields["swap_kb"] = service.System.Swap.Kilobyte
+			fields["swap_percent"] = service.System.Swap.Percent
+			acc.AddFields("system", fields, tags)
+		} else if service.Type == "6" {
+			fields["permissions"] = service.Mode
+			acc.AddFields("fifo", fields, tags)
+		} else if service.Type == "7" {
+			fields["last_started_time"] = service.Program.Started * 1000
+			fields["program_status"] = service.Program.Status
+			fields["output"] = service.Program.Output
+			acc.AddFields("program", fields, tags)
+		} else if service.Type == "8" {
+			fields["link_state"] = service.Link.State
+			fields["link_speed"] = service.Link.Speed
+			fields["link_mode"] = linkMode(service)
+			fields["download_packets_now"] = service.Link.Download.Packets.Now
+			fields["download_packets_total"] = service.Link.Download.Packets.Total
+			fields["download_bytes_now"] = service.Link.Download.Bytes.Now
+			fields["download_bytes_total"] = service.Link.Download.Bytes.Total
+			fields["download_errors_now"] = service.Link.Download.Errors.Now
+			fields["download_errors_total"] = service.Link.Download.Errors.Total
+			fields["upload_packets_now"] = service.Link.Upload.Packets.Now
+			fields["upload_packets_total"] = service.Link.Upload.Packets.Total
+			fields["upload_bytes_now"] = service.Link.Upload.Bytes.Now
+			fields["upload_bytes_total"] = service.Link.Upload.Bytes.Total
+			fields["upload_errors_now"] = service.Link.Upload.Errors.Now
+			fields["upload_errors_total"] = service.Link.Upload.Errors.Total
+			acc.AddFields("network", fields, tags)
+		}
+	}
+	return nil
+}
+
+func linkMode(s Service) string {
+	if s.Link.Duplex == 1 {
+		return "Duplex Mode"
+	} else {
+		return "Simplex Mode"
+	}
+}
+
+func serviceStatus(s Service) string {
+	var status string
+
+	if s.MonitoringStatus == 0 || s.MonitoringStatus == 2 {
+		status = monitoringStatus(s)
+	} else if s.Status == 0 {
+		status = "Running"
+	} else {
+		status = "Failure"
+	}
+
+	if s.PendingAction > 0 {
+		if s.PendingAction >= len(pendingActions) {
+			return fmt.Sprintf("%s - pending", status)
+		}
+		status = fmt.Sprintf("%s - %s pending", status, pendingActions[s.PendingAction])
+	}
+	return status
+}
+
+func monitoringMode(s Service) string {
+	switch s.MonitorMode {
+	case 0:
+		return "Monitoring in Active mode"
```

Comment:
I suggest we make these strings consistent with the Monit web interface, in mine:
```
Monitoring status:  Monitored
Monitoring mode:    active
```

We should try to be consistent like this in any string conversions.

Reply:
Done. Please check